### ISSUE TITLE
Can't compile file

### ISSUE THREAD

**Issue by chaniwat** [platformio/platformio-atom-ide#478]
![error](https://cloud.githubusercontent.com/assets/3206134/26205765/f4ff8f22-3c0c-11e7-8488-2da2aa444f75.png)

**Comment by ivankravets** [comment]
Try to remove `~/.platformio` folder and restart build.

**Comment by chaniwat** [comment]
I reinstall everything (atom, platformio-ide, python) but it not solved.
run on windows 10

in platformio.ini
![image](https://cloud.githubusercontent.com/assets/3206134/26207617/45e403a0-3c12-11e7-8964-ff95328e88a3.png)

project structue
![image](https://cloud.githubusercontent.com/assets/3206134/26207677/69fee232-3c12-11e7-951d-ebe9979db0cf.png)

**Comment by chaniwat** [comment]
Update, try to run outside atom

Can run on command line
![image](https://cloud.githubusercontent.com/assets/3206134/26209543/4e6f7626-3c17-11e7-9394-78dcc3b1f8b9.png)

But can't run on powershell
![image](https://cloud.githubusercontent.com/assets/3206134/26210253/3f2b3270-3c19-11e7-9844-e05b11bc7a2c.png)

**Comment by ivankravets** [comment]
Please provide an output of `echo %PATH`

**Comment by chaniwat** [comment]
```
C:\Python27\;C:\Python27\Scripts;C:\Program Files\Docker\Docker\Resources\bin;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files(x86)\Ampps\php;C:\ProgramData\ComposerSetup\bin;C:\Program Files\Git\cmd;C:\ansicon;C:\Program Files (x86)\Microsoft Emulator Manager\1.0\;C:\Users\Meranote\AppData\Local\Android\sdk\platform-tools;C:\Program Files (x86)\GtkSharp\2.12\bin;C:\Users\Meranote\AppData\Roaming\Composer\vendor\bin;C:\Users\Meranote\AppData\Local\atom\bin;C:\Program Files\nodejs\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\dotnet\;C:\Program Files\Java\jre1.8.0_102\bin;C:\Program Files\Java\jdk1.8.0_102\bin;C:\apache-maven-3.3.9\bin;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\PuTTY\;C:\Users\Meranote\AppData\Roaming\npm;C:\Users\Meranote\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin;C:\Program Files (x86)\Ampps\mongodb\bin;C:\chromedriver_win32;C:\Program Files\LLVM\bin;C:\Users\Meranote\AppData\Local\atom\bin
```

**Comment by chaniwat** [comment]
I found the problem that cause error is "ansicon", i uninstall it and its work now.
(compile/upload via IDE and command line (powershell))

![image](https://user-images.githubusercontent.com/3206134/27766600-b7db26ea-5eff-11e7-8231-494dab15bd81.png)

I installed ansicon as autorun, so everytime i open cmd (or powershell) it hook into it. I think the problem is ansicon, not compatible with powershell (because everytime i try to run pio command via powershell it cause an error but on cmd it work normally). So i uninstall it and it work. 😄
